proposals/0148-stake-program-move-instructions.md

```diff
@@ -0,0 +1,164 @@
+---
+simd: '0148'
+title: MoveStake and MoveLamports Instructions
+authors:
+  - Hanako Mumei
+category: Standard
+type: Core
+status: Draft
+created: 2024-04-30
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+We propose introducing two new instructions to the stake program for moving
+value between stake accounts with identical `Authorized` and `Lockup`:
+
+* `MoveStake`: Move a given `amount` of active stake from one fully active
```

Comment:
nit: Someone else might bring this up, but we can probably ditch the term "fully" and just say "active" everywhere. Stakes can be active, inactive, activating, or deactivating. We could even just combine "activating or deactivating" and say "transient".

Reply:
hmm i was trying to be unambigous because a stake account caught by the (de)activation warmup is partly "(de)activating" and partly "active." `Split` for instance treats any account with nonzero effective stake as "active" when determining if it can fund destination rent

Reply:
Ah I see what you mean... for the sake of documents, we might want to assume that "active" == "fully active" and not let the code's treatment of "active" create confusion.